meetup/api.py

```diff
@@ -162,6 +162,8 @@ def _call(self, service_name, parameters=None, **kwargs):
             raise exceptions.HttpUnauthorized
         if response.status_code == 404:
             raise exceptions.HttpNotFoundError
+        if response.status_code == 503:
+            raise exceptions.HttpServerError
```

Comment:
`exceptions.HttpServerError` is a base class for `5xx` errors, not meant to be raised directly; similar to `exceptions.HttpClientError`.

If we are to raise an exception for `5xx`, create a new class (based from `HttpServerError`), then raise that.